src/JsonLd/ContextBuilder.php

```diff
@@ -14,6 +14,7 @@
 namespace ApiPlatform\Core\JsonLd;
 
 use ApiPlatform\Core\Api\UrlGeneratorInterface;
+use ApiPlatform\Core\Hydra\Serializer\CollectionNormalizer;
```

Comment:
The normalizer can depend of this class, but not the other way. You should move the main constant here IMHO.

Reply:
Actually, we usually don't use constants for attributes, so just use the raw string.